### PR TITLE
Polkadot Protocol > Architecture > System Chains > BridgeHub

### PR DESCRIPTION
**Pull request by CrackTheCode016**
Adds high level documentation of the functionality and components that make up the BridgeHub system chain

### CODE REVIEW

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:9]:
```suggestion

The Bridge Hub system chain is responsible for providing the necessary functionality for the relay chain (and parachains) to enable trustless bridging between chains like Polkadot, Kusama, and Ethereum. It includes a set of [pallets](../../glossary.md#pallet) which facilitate the sending and receiving of messages, implementations of BEEFY, GRANDPA, and Ethereum light clients to follow and verify the state of both the source and target chain. BridgeHub also provides XCM support for target and source chains, provided XCM is implemented on the target chain.
```

Comment (nhussein11) [reply polkadot-protocol/architecture/system-chains/bridge-hub.md:9]: I'm wondering if it would be better to add the following reference for the term "a set of pallets": https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs#L517

Or maybe something more "human-readable" haha, but I think you get my point

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:13]: We should stick to either "BridgeHub" or "Bridge Hub". IMO, "Bridge Hub" would be better since we also have "Asset Hub" in the same format

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:15]:
```suggestion
[BEEFY (Bridge Efficiency Enabling Finality Yielder)](https://wiki.polkadot.network/docs/learn-consensus#bridging-beefy){target=\_blank} is instrumental in this solution. It provides a more efficient way to verify the consensus on the relay chain. It allows the participants in a network to verify finality proofs, meaning a remote chain like Ethereum can verify the state of Polkadot at a given block height.
```

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:31]: Just formatting stuffs to help the formatting team :)

```suggestion

- [Pallet Bridge GRANDPA](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_grandpa/index.html){target=\_blank} - an on-chain GRANDPA light client for Substrate based chains
- [Pallet Bridge Parachains](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_parachains/index.html){target=\_blank} - a finality module for parachains
- [Pallet Bridge Messages](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_grandpa/index.html){target=\_blank} - a pallet which allows sending, receiving, and tracking of inbound and outbound messages 
- [Pallet XCM Bridge](https://paritytech.github.io/polkadot-sdk/master/pallet_xcm_bridge_hub/index.html){target=\_blank} - a pallet which, with the Bridge Messages pallet, adds XCM support to bridge pallets
```

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:41]: Again, just formatting stuff to help the team

```suggestion
- [Snowbridge](https://wiki.polkadot.network/docs/learn-snowbridge){target=\_blank} - a general-purpose, trustless bridge between Polkadot and Ethereum
- [Hyperbridge](https://wiki.polkadot.network/docs/learn-hyperbridge){target=\_blank} - a cross-chain solution built as an interoperability coprocessor, providing state-proof-based interoperability across all blockchains
- [Polkadot <> Kusama Bridge](https://wiki.polkadot.network/docs/learn-dot-ksm-bridge){target=\_blank} - a bridge that utilizes relayers to bridge the Polkadot and Kusama relay chains trustlessly
```

**Comment by nhussein11** [comment]
Besides those details, it LGTM! 🚀  Just tag me when you work on the comments so that I can approve the PR on our end :)

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (CrackTheCode016) [reply polkadot-protocol/architecture/system-chains/bridge-hub.md:35]: These are basically a few different pallets - should I link that here @nhussein11? Or link: https://github.com/paritytech/polkadot-sdk/tree/aff3a0796176ff3c0ee1b89c2f1d811a858f17a8/bridges/snowbridge/pallets

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (CrackTheCode016) [reply polkadot-protocol/architecture/system-chains/bridge-hub.md:9]: Maybe the Subscan link? https://bridgehub-polkadot.subscan.io/runtime

Comment (nhussein11) [reply polkadot-protocol/architecture/system-chains/bridge-hub.md]: yeah i think that works 👍

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (nhussein11) [reply polkadot-protocol/architecture/system-chains/bridge-hub.md]: https://github.com/paritytech/polkadot-sdk/tree/aff3a0796176ff3c0ee1b89c2f1d811a858f17a8/bridges/snowbridge/pallets

does the trick i think, let's add that one better, thank you!

**Comment by CrackTheCode016** [comment]
@nhussein11 thank you, should be good to go!

**Review by nhussein11 — APPROVED** [review]
niceee! 💯

**Review by dawnkelly09 — COMMENTED** [review]
I did the grammar and formatting updates so I will tag Erin in for an approval from our team. Also tested links to ensure they work, use absolute links, and have the {target=\_blank}

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (eshaben) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:32]:
```suggestion
- [**Pallet Bridge GRANDPA**](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_grandpa/index.html){target=\_blank} - an on-chain GRANDPA light client for Substrate based chains
- [**Pallet Bridge Parachains**](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_parachains/index.html){target=\_blank} - a finality module for parachains
- [**Pallet Bridge Messages**](https://paritytech.github.io/polkadot-sdk/master/pallet_bridge_messages/index.html){target=\_blank} - a pallet which allows sending, receiving, and tracking of inbound and outbound messages 
- [**Pallet XCM Bridge**](https://paritytech.github.io/polkadot-sdk/master/pallet_xcm_bridge_hub/index.html){target=\_blank} - a pallet which, with the Bridge Messages pallet, adds XCM support to bridge pallets
```

File: polkadot-protocol/architecture/system-chains/bridge-hub.md
Comment (eshaben) [inline polkadot-protocol/architecture/system-chains/bridge-hub.md:44]:
```suggestion
- [**Snowbridge**](https://wiki.polkadot.network/docs/learn-snowbridge){target=\_blank} - a general-purpose, trustless bridge between Polkadot and Ethereum
- [**Hyperbridge**](https://wiki.polkadot.network/docs/learn-hyperbridge){target=\_blank} - a cross-chain solution built as an interoperability coprocessor, providing state-proof-based interoperability across all blockchains
- [**Polkadot <> Kusama Bridge**](https://wiki.polkadot.network/docs/learn-dot-ksm-bridge){target=\_blank} - a bridge that utilizes relayers to bridge the Polkadot and Kusama relay chains trustlessly
```

**Review by eshaben — COMMENTED** [review]
Nice work! Looks good!

**Comment by nhussein11** [comment]
@polkadot-developers/docs-formatting-team this needs one more approval on your side, it's good to go 🚀

**Pull request merged**
